src/test/java/com/softserve/academy/spaced/repetition/service/AuditServiceImplTest.java

```diff
@@ -0,0 +1,168 @@
+package com.softserve.academy.spaced.repetition.service;
+
+import com.softserve.academy.spaced.repetition.domain.Audit;
+import com.softserve.academy.spaced.repetition.repository.AuditRepository;
+import com.softserve.academy.spaced.repetition.service.impl.AuditServiceImpl;
+import com.softserve.academy.spaced.repetition.util.DomainFactory;
+import com.softserve.academy.spaced.repetition.utils.audit.AuditingAction;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+@Transactional
+public class AuditServiceImplTest {
+
+    @InjectMocks
+    private AuditServiceImpl auditService;
+
+    @Mock
+    private AuditRepository auditRepository;
+
+    private Audit audit;
+
+    @Before
+    public void setUp() {
+        final long AUDIT_ID = 1L;
+        final String EMAIL = "account@test.com";
+        final String IP = "1.1.1.1";
+        final String ROLE = "ROLE_USER";
+        AuditingAction action;
+        action = AuditingAction.VIEW_ALL_COMMENTS_FOR_COURSE;
+        audit = DomainFactory.createAudit(AUDIT_ID, EMAIL, action, new Date(), IP, ROLE);
+    }
+
+    private List<Audit> createAuditList() {
+        List<Audit> auditList = new ArrayList<>();
+        auditList.add(audit);
+        return auditList;
+    }
+
+    @Test
+    public void testGetFullAuditList() {
+        when(auditRepository.findAll()).thenReturn(createAuditList());
```

Comment:
if you use `createAuditList` almost in every method, maybe will be better create some variable